packages/core/package.json

```diff
@@ -3,7 +3,8 @@
   "version": "1.0.0-alpha.13",
   "description": "UMA smart contracts and unit tests",
   "dependencies": {
-    "@umaprotocol/common": "^1.0.0-alpha.13"
+    "@umaprotocol/common": "^1.0.0-alpha.13",
+    "cli-progress": "^3.8.2"
```

Comment:
nit: why did this need to be added? And does it need to be a regular dep or can it be a dev dep? Right now, everything used by the scripts in `packages/core/scripts` are considered devDependencies since those scripts are not exported as a part of the npm package.

Reply:
This dependency was somehow lost during the most recent Lerna re-structure and the script failed to run without it (it gives a pretty progress bar as the script runs :) ). Not sure why but it was not present anymore. I can move it to a dev dependency though.

Reply:
Gotcha, makes sense. SGTM!